easybuild/easyconfigs/m/MySQL/MySQL-5.6.26-intel-2015b-clientonly.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'MySQL'
+version = '5.6.26'
+versionsuffix = '-clientonly'
+
+homepage = 'http://www.mysql.com/'
+description = """MySQL is (as of March 2014) the world's second most widely used
+ open-source relational database management system (RDBMS)."""
+
+source_urls = ['http://dev.mysql.com/get/Downloads/MySQL-%(version_major_minor)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+dependencies = [
+    ('libevent', '2.0.22'),
+    ('libreadline', '6.3', '', ('GNU', '4.9.3-2.25')),
```

Comment:
I don't see [an EasyBuild config for libreadline-6.3-GCC-4.9.3](https://github.com/hpcugent/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/l/libreadline). Should this be in the PR?